docs/documentation/writing-documentation.md

```diff
@@ -31,7 +31,16 @@ meet the following requirements:
 - Installations of the following tools and languages:
     - [git](https://git-scm.com/)
     - [Python](https://www.python.org/)
-    - [pip](https://pip.pypa.io/en/stable/installing/) (comes by default with Python 2 >= 2.7.9 or Python 3 >= 3.4)
+    - [pip](https://pip.pypa.io/en/stable/installing/) (usually comes by default with Python 2 >= 2.7.9 or Python 3 >= 3.4)
+
+!!!note
+    On macOS, the OS-distributed Python does not come with pip.
+    Run the following to install it:
+
+        :::console
+        $ sudo easy_install pip
```

Comment:
This seems like it's missing a step. Where does `easy_install` come from?

Reply:
I'm pretty sure that one does come with the OS Python.

Reply:
@osg-cat could you verify that you have `easy_install` available through the OS Python installation?

Reply:
Tims-MacBook-Pro: which easy_install
/usr/bin/easy_install